pkg/storaged/format-dialog.jsx

```diff
@@ -484,17 +487,19 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                         mount_options.push(vals.mount_options.extra);
 
                     mount_point = vals.mount_point;
-                    if (mount_point[0] != "/")
-                        mount_point = "/" + mount_point;
-
-                    config_items.push(["fstab", {
-                        dir: { t: 'ay', v: utils.encode_filename(mount_point) },
-                        type: { t: 'ay', v: utils.encode_filename("auto") },
-                        opts: { t: 'ay', v: utils.encode_filename(mount_options.join(",") || "defaults") },
-                        freq: { t: 'i', v: 0 },
-                        passno: { t: 'i', v: 0 },
-                        "track-parents": { t: 'b', v: true }
-                    }]);
+                    if (mount_point != "") {
+                        if (mount_point[0] != "/")
+                            mount_point = "/" + mount_point;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18742-20230508-073603-1749d05b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)